lib/mix/lib/mix/tasks/run.ex

```diff
@@ -40,11 +40,14 @@ defmodule Mix.Tasks.Run do
 
       elixir --sname hello -S mix run --no-halt
 
+  This task is automatically reenabled, so it can be called multiple times
+  with different arguments.
+
   ## Command-line options
 
     * `--config`, `-c`  - loads the given configuration file
     * `--eval`, `-e` - evaluates the given code
-    * `--require`, `-r` - requires pattern before running the command
+    * `--require`, `-r` - executes the given pattern/file
```

Comment:
I wonder if this change is consistent with the description of the next flag:

> `parallel`, `-p` - makes all requires parallel

Maybe “requires (and executes) the given pattern/file”?

Reply:
I think so, since the word "requires" is talking about the `--require` flag. Thoughts?